### ISSUE TITLE
Null String concatenation same as empty string

### ISSUE THREAD

**Issue by luss** [IvorySQL/IvorySQL#45]
## Enhancement
 This is important in Oracle Compatability mode

**Comment by BingAo-Office** [comment]
Although this feature is easy to implement, some extension and functions can cause inexplicable errors if this feature is implemented. Therefore, the scope about  implementation and parameter control is the focus of the implementation of this feature.

**Comment by luss** [comment]
yeah...   perhaps there needs to be a separate GUC for this beyond the
major GUC for Oracle compatible mode.

**Comment by OreoYang** [comment]
> It looks like this is implemented:

Yes, it was implemented in PR https://github.com/IvorySQL/IvorySQL/pull/240
